pinc/project_edit.inc

```diff
@@ -6,26 +6,17 @@ include_once($relPath.'project_states.inc');
 
 function user_can_add_project_pages($projectid, $page_type = "normal")
 {
-    global $site_supports_metadata;
-
     $project = new Project($projectid);
     $state = $project->state;
 
     if ($page_type == "tp&v") { // TP&V = Title Page & Verso
```

Comment:
As far as I can tell (and I admit that I may be wrong about this) `tp&v` and `tpnv` are both specific to the metadata code. There is a whole section of [`do_early_uploads`](https://github.com/DistributedProofreaders/dproofreaders/blob/443817fcb4c03afaf5aea273f412325588f4d8ef/project.php#L877) in `project.php` that deals only with uploading title page and verso that, as far as I can recall, I've never seen displayed in the project load form when loading files into a project (but I'm not smart enough to figure out why it's not displaying).

There's also a bunch of code in `add_files.php` that deals with loading TP&V. As part of that section of code, the project state is updated to `project_new_waiting_app`. (The bits here, and in `add_files.php` and `project.php` are the three places I've found that refer to loading TP&V; both `tpnv` and `tp&v` are used.)

Reply:
Yup, indeed all the of `tp&v` code needs to come out too, you are correct. Thanks for catching that.

Reply:
I finally remembered to create a test project and everything went as expected.